thunder/transforms/quantization_cpu.py

```diff
@@ -0,0 +1,224 @@
+# NOTE: The code for CPU quantization in this file has been adapted from a not-yet-merged branch of the
```

Comment:
Can you please add the original copyright notice and a link to the license?
(also, referring to permalinks instead of branches is a bit more reliable)